crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py

```diff
@@ -802,14 +641,7 @@ def stats_generate_handler(args: argparse.Namespace):
 
                         s3_data_object["events"] = events_data
 
-                        s3_data_object["generic"] = generate_metrics(
-                            db_session=db_session,
-                            blockchain_type=blockchain_type,
-                            address=address,
-                            timescale=timescale,
-                            metrics=generic_metrics_names,
-                            start=start_date,
-                        )
+                        s3_data_object["generic"] = {}
```

Comment:
Does this break the frontend?

Reply:
No not breaking  dict leght must be more then 0
```javascript 
  const genericKeys = useMemo(
    () =>
      Object.keys(data?.generic ?? {}).length > 0
        ? Object.keys(data?.generic)
        : undefined,
    [data]
  );
{data?.generic && genericKeys && (
```
It mainly the case when you not select any checkboxes)

Reply:
Cool. Approving.